compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -2596,8 +2596,16 @@ setTranslationInfoAndRootConfig(func::FuncOp entryPointFn,
     }
 
     // Set vector level tile sizes for other operations individually.
+    // We allow vmvx with micorkernels having failures because it is okay to not
+    // scale tile sizes. The configuration has dynamic inner tile sizes. It
+    // fails only if the compute op is tensor.pack and inner tile sizes are
+    // dynamic.
```

Comment:
I'm working on some refactor works of tile size selection so I have a related question:

If the inner tile sizes are dynamic and we can't scale the tile sizes on outer dims, does that mean we will potentially tile with too large tile sizes?

Reply:
Yes, we will potentially tile with too large tile sizes.

Reply:
There are very few cases at this moment. It is only used by VMVX with data-tiling on + microkernels. I think we should split the file properly, so we can make vmvx and llvm-cpu more isolated. We can move the logic out of `LLVMCPU/KernelDispatch.cpp` after refactoring; I can help on that. What do you think?

Reply:
SGTM